horovod/runner/elastic/worker.py

```diff
@@ -30,8 +31,9 @@
 
 class HostsUpdatedRequest(object):
     """Notifies worker that the set of available hosts/slots has changed."""
-    def __init__(self, timestamp):
+    def __init__(self, timestamp, shrink=False):
```

Comment:
Let's send the actual `HostUpdateResult` so this API becomes more flexible in the future.